### ISSUE TITLE
Duvida - Registrado

### ISSUE THREAD

**Issue by soldadowc** [Trust-Code/python-boleto#35]
Consigo gerar boletos registrados com essa biblioteca?
Estou abrindo um empresa e não quero ficar pagando esses sites para gerar o boleto de cobrança. vou cria um script para gerar quando for necessário.

**Comment by pfrozi** [comment]
Não, ela só gera o boleto impresso. Mas atualmente o registro online é bem simples de implementar na maior parte dos bancos. Com qual banco tu pretendes trabalhar?
